install_template/templates/products/migration-toolkit/base.njk

```diff
@@ -4,7 +4,5 @@
 {% endblock prodprereq %}
 {% block postinstall %}
 ## Initial configuration
-Before invoking Migration Toolkit, you must download and install a freely available source-specific driver. To download a driver, or for a link to a vendor download site, see the [Third Party JDBC Drivers](https://www.enterprisedb.com/software-downloads-postgres#third-party-jdbc-drivers) on the Downloads page.
-
-After downloading the source-specific driver, move the driver file into the `<mtk_install_dir>/lib` directory.
-{% endblock postinstall %} 
\ No newline at end of file
+Before invoking Migration Toolkit, you must download and install a JDBC driver. See [Installing a JDBC driver](/migration_toolkit/latest/05_installing_mtk/installing_jdbc_driver/) for details.
```

Comment:
Actually, in most cases you will need to install more than one JDBC driver, one for the source database and one for the target database.  Perhaps this should read, 

> Before invoking Migration Toolkit, you must download and install JDBC drivers for connecting to the source and target databases.

Where does this text/file appear in the documentation?  It has an extension (.njk) I'm not familiar with.

Reply:
Yes, the base file is part of the template framework.